packages/sdk/src/health_indicator/health_indicator.ts

```diff
@@ -19,20 +20,32 @@ interface IHealthIndicator {
 }
 
 export class HealthIndicator implements IHealthIndicator {
+  private isStarted = false;
+
   private readonly libp2p: Libp2p;
   private readonly events: IWakuEventEmitter;
 
   private value: HealthStatus = HealthStatus.Unhealthy;
+  private readonly debouncedAssessHealth: ReturnType<typeof debounce>;
 
   public constructor(params: HealthIndicatorParams) {
     this.libp2p = params.libp2p;
     this.events = params.events;
 
     this.onPeerIdentify = this.onPeerIdentify.bind(this);
     this.onPeerDisconnected = this.onPeerDisconnected.bind(this);
+
+    this.debouncedAssessHealth = debounce(() => {
+      void this.assessHealth();
+    }, 100);
```

Comment:
nit: instead of hardcoding `100` use a constant var with a name indicating what this value means e.g. `DEBOUNCE_MS`